src/subcommand/server.rs

```diff
@@ -1172,6 +1184,37 @@ impl Server {
     })
   }
 
+  async fn inscription_children(
+    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path(inscription_id): Path<InscriptionId>,
+  ) -> ServerResult<Response> {
+    Self::inscription_children_from_pages(
+      Extension(page_config),
+      Extension(index),
+      Path((inscription_id, 0)),
+    )
+    .await
+  }
+
+  async fn inscription_children_from_pages(
+    Extension(page_config): Extension<Arc<PageConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path((inscription_id, page_index)): Path<(InscriptionId, usize)>,
+  ) -> ServerResult<Response> {
+    let _inscription = index
+      .get_inscription_by_id(inscription_id)?
+      .ok_or_not_found(|| format!("inscription {inscription_id}"))?;
+
+    let children = index.get_children_by_inscription_id(inscription_id)?;
```

Comment:
Instead of getting all children, let's try to only get only the number we need (100). Have a look how we do the inscriptions pagination, specifically `fn inscriptions_inner()` in `server.rs` and `fn get_latest_inscriptions_with_prev_and_next` in `index.rs`. You will have to use the `SEQUENCE_NUMBER_TO_INSCRIPTION_ID` table to get the children in the right order since the redb multimap is unordered (at least in the sense we care about). If this is too daunting we can also show this/do it together on the coding stream tomorrow.

Reply:
Would be great to do this on the stream!